server/webapp/WEB-INF/rails.new/app/assets/javascripts/build_output_observer.js

```diff
@@ -72,15 +72,21 @@ BuildOutputObserver.prototype = {
 
     _update_live_output_raw: function (build_output) {
         var is_output_empty = !build_output;
-        var buildoutputPreElement = $('buildoutput_pre');
+        // we loop through these elements below, because there's 2 of those
+        // on on the console tab, and another on failures tab
+        var buildoutputPreElement = $$('.buildoutput_pre');
         if (!is_output_empty && buildoutputPreElement) {
             var escapedOutPut = build_output.escapeHTML();
             if (Prototype.Browser.IE) {
                 // Fix for the IE not wrap /r in pre bug
                 escapedOutPut = '<br/>' + escapedOutPut.replace(/\n/ig, '<br\/>');
-                buildoutputPreElement.innerHTML += escapedOutPut;
+                buildoutputPreElement.each(function(e, i){
+                    e.innerHTML += escapedOutPut;
```

Comment:
That block of code uses prototype, not jQuery.

Also, I was not willing to make bigger changes there knowing that this is old code that is going to be superseded by `_update_live_output_color`.